django/api/models/go_electric_rebate.py

```diff
@@ -9,19 +9,22 @@
 )
 
 from django.core.validators import MinLengthValidator
-
+from django.conf import settings
 from django_extensions.db.models import TimeStampedModel
 from django.utils.translation import gettext_lazy as _
 from api.models.go_electric_rebate_application import GoElectricRebateApplication
 
 
 class GoElectricRebate(TimeStampedModel):
-    id = AutoField(primary_key=True)
+    user = ForeignKey(

```

Comment:
Do we really need the user on this model? It can be accessed via the application.

Reply:
i was not able to insert data from admin panel without user field

Reply:
I'm able to use this in the admin panel without the user field. This should be removed plz.